cloudhub-2/modules/ROOT/pages/ch2-deploy-api.adoc

```diff
@@ -0,0 +1,180 @@
+
+The following example shows the request to deploy a new app using a TCP port mapping:
+
+[source,json]
+----
+POST https://anypoint.mulesoft.com/amc/application-manager/api/v2/organizations/{organizationId}/environments/{environmentId}/deployments
+
+{
+ "name": "logs-app-uc01",
+ "labels": [
+   "beta"
+ ],
+ "target": {
+   "provider": "MC",
+   "targetId": "cfe2157f-438b-4fb8-81fe-13ec89c55545",
+   "deploymentSettings": {
+     "clustered": false,
+     "enforceDeployingReplicasAcrossNodes": false,
+//------------------------------------NEW-------------------------------------
+     "tcp": {
+         "inbound": {
+           "ports": [
+             {
+               "portNumber": 30500,
+               "applicationPortNumber": 9000  
+             },
+             {
+               "portNumber": 30501,
+               "applicationPortNumber": 9001  
+             },
+             {
+               "portNumber": 30502,
+               "applicationPortNumber": 9002  
+             },
+           ]
+         }
+       },
+//------------------------------------NEW-------------------------------------
+     "jvm": {},
+     "runtimeVersion": "4.4.0:20220523-1",
+     "updateStrategy": "rolling",
+     "disableAmLogForwarding": false,
+     "generateDefaultPublicUrl": true
+   },
+   "replicas": 1
+ },
+ "application": {
+   "ref": {
+     "groupId": "66310c16-bce5-43c4-b978-5945ed2f99c5",
+     "artifactId": "app-with-40-schedulers",
+     "version": "1.0.5",
+     "packaging": "jar"
+   },
+   "assets": [],
+   "desiredState": "STARTED",
+   "configuration": {
+     "mule.agent.application.properties.service": {
+       "applicationName": "logs-app-uc01",
+       "properties": {},
+       "secureProperties": {}
+     },
+     "mule.agent.logging.service": {
+       "artifactName": "logs-app-uc01",
+       "scopeLoggingConfigurations": []
+     }
+   },
+   "integrations": {
+     "services": {
+       "objectStoreV2": {
+         "enabled": false
+       }
+     }
+   },
+   "vCores": "0.05"
+ }
+}
+----
+
+Deployments that include unavailable ports fail. Successful deployments use the newly mapped ports, and those ports become unavailable. If the app is deleted, the ports become available again.
+
+For incorrect port configurations, the following errors appear:
+
+* `App <name> has leased the maximum number of ports.` occurs when you specify more than 10 ports.
+* `Ports must be within the supported range of 30500 - 32500` occurs when the port you specify is out of range.
+* `Port 30501 is already in use by other owner` occurs when the port you specify is in use by another app.
```

Comment:
I dont think, we should show the error message that will be thrown , since those are maintained in the code and its not ideal to keep a track in the documentation. If we ever plan to improve/enhance on error messaging, we would have to fix it in docs as well. Let me know your opinion on this. CC: @afelisatti @vli63127